mitosheet/src/types.tsx

```diff
@@ -825,8 +825,9 @@ export interface MitoStateUpdaters {
     setUIState: React.Dispatch<React.SetStateAction<UIState>>,
 }
 
-export interface CSVExportState { exportType: 'csv' }
-export interface ExcelExportState { exportType: 'excel', sheetIndexes: number[] }
+export interface ExportState { fileName?: string, exportType: 'csv' | 'excel' }
+export interface CSVExportState extends ExportState { exportType: 'csv' }
+export interface ExcelExportState extends ExportState { exportType: 'excel', sheetIndexes: number[] }
```

Comment:
These are pleasant types! We rarely use extends, but just b/c we forget about them. Nice!

Reply:
Thanks!